models/articles.js

```diff
@@ -44,14 +44,20 @@ module.exports = (sequelize, DataTypes) => {
       onDelete: 'CASCADE',
       onUpdate: 'CASCADE'
     }
-  }, {});
-  Articles.associate = (model) => {
-    Articles.hasMany(model.Reactions, { as: 'reactions', foreignKey: 'articleId' });
+  });
+  Articles.associate = (models) => {
+    Articles.hasMany(models.Reactions, { as: 'reactions', foreignKey: 'articleId' });
+    Articles.hasMany(models.Bookmark, {
+      foreignKey: 'articleSlug',
+      as: 'bookmarked',
+      targetKey: 'articleSlug',
+      sourceKey: 'slug',
+    });
   };
   SequelizeSlugify.slugifyModel(Articles, {
     source: ['title'],
     slugOptions: { lower: true },
-    overwrite: true,
+    overwrite: false,
     column: 'slug'
```

Comment:
Hi Afeez, I think we should hold off on this change pending feedback from the TTL.